CurrenyConversion.jsx

```diff
@@ -4,10 +4,12 @@ es6Promise.polyfill();
 import fetch from 'isomorphic-fetch'
 
 class CurrenyConversion extends React.Component {
-  constructor () {
-    super()
-    this.state = { val: 0 }
-  }
+  static propTypes = {
+     from: React.PropTypes.string.isRequired,
+     to: React.PropTypes.string.isRequired,
+     val: React.PropTypes.string.isRequired   
+  };
+  state = { val: 0 };
```

Comment:
@hemanth  its experimental and guess stage 0 in babel ...plus not a confirmed spec I guess ..SORRY !